app/src/main/java/org/mozilla/reference/browser/BrowserTestActivity.kt

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.reference.browser
+
+import android.os.Bundle
+import mozilla.components.support.utils.SafeIntent
+import org.mozilla.reference.browser.ext.testConfig
+
+/**
+ * This activity is used for performance testing with Raptor/tp6:
+ * https://wiki.mozilla.org/Performance_sheriffing/Raptor
+ */
+class BrowserTestActivity : BrowserActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        testConfig = SafeIntent(intent).extras
```

Comment:
nit: tbh I'd just use the same activity and do that conditional for a specified intent.action.

Reply:
We discussed this offline, and a separate activity is fine, so we don't clutter our main code path.